client/my-sites/domains/domain-management/domain-overview-pane/style.scss

```diff
@@ -0,0 +1,7 @@
+@media (max-width: 960px) {
```

Comment:
What do you think of using the [media query mixins](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md#media-queries) here instead?

Reply:
Done it here https://github.com/Automattic/wp-calypso/pull/97193/commits/ddd028a0cb3a9c56036dda1335b93b61666b433b